src/main/external-resources/web/bump/bump.js

```diff
@@ -33,71 +33,71 @@ var bump = (function() {
 	}
 
 	function hookup() {
-		$('#bplaylist').mousedown(function(){bump.getPlaylist(1);})
+		jQuery('#bplaylist').mousedown(function(){bump.getPlaylist(1);})
 			.click(function(){bump.edited(1);})
 			.blur(function(){bump.edited(0);});
-		$('#brenderers').change(function(){bump.setRenderer();});
-		$('#bumpvol').attr('type','range').attr('max',100);
-		$('#bumpvol').keyup(function(){bump.sliding(false);})
+		jQuery('#brenderers').change(function(){bump.setRenderer();});
```

Comment:
Why use jquery instead of $?

Reply:
I used a new jquery library that doesn't recognize the $ symbol, so I changed $ by jQuery. That was why.
I will try to fit to what is done, not to create too much conflicts.